src/common/interfaces.ts

```diff
@@ -94,3 +77,14 @@ export interface Tree {
   kennzeich?: string | null;
   caretaker?: string | null;
 }
+
+type TreeId = string;
```

Comment:
Shouldn't that be in common/types.ts

Reply:
I used it to indicate what the string is about. It shouldn't be anywhere else though.